### DISCUSSION TITLE
Modeless panes page images seem incomplete

### DISCUSSION THREAD

**Discussion by QGerrit** [RocketCommunicationsInc/astro#956] [General]
https://www.astrouxds.com/patterns/modeless-panes/

I feel like something's being pointed to that I'm missing ;)

![image](https://user-images.githubusercontent.com/97202638/204357009-4ea5b425-22ae-4463-8953-2376f69fe52e.png)

**Comment by FMorrison87** [comment]
Hello! These images are indeed incomplete. I am working with our design team to get updated versions and to have them published to the website. I will update this discussion once those images go live. Thanks very much for bringing this to our attention.

**Comment by FMorrison87** [comment]
@QGerrit This issue has now been fixed and is live on our website: https://www.astrouxds.com/patterns/modeless-panes/

Thanks again for pointing this out!

**Reply by QGerrit** [reply]
Awesome, thanks!
